### ISSUE TITLE
Does this module work with @import and other vars with SuitCSS?

### ISSUE THREAD

**Issue by priyatam** [aaronj1335/rework-webpack-loader#4]
@remoe asked [the same question](https://github.com/webpack/css-loader/issues/28) and @sokra pointed him to your library. Running webpack based on the config in your README throws an error:

```
Module not found: Error: Cannot resolve 'file' or 'directory' ./suitcss in /
~/rework-samples
 @ ./~/rework-webpack-loader!./index.css 1:0-20
```

I've tried adding [extract-text](https://github.com/webpack/extract-text-webpack-plugin) and could not get it to work with rework plugins.

sample css:

``` css
@import "suitcss";

:root {
  --base-link-color: #069;
}
```

sample js:

``` js
var css = require("./index.css");
```

webpack.config.js:

``` js

var path = require('path');
var pkg = require('./package.json');
var util = require('util');
var webpack = require('webpack');
var ExtractTextPlugin = require('extract-text-webpack-plugin');
var reworkLoader = require('rework-webpack-loader');
var reworkVars = require('rework-vars');
var reworkCalc = require('rework-calc');
var reworkCustomMedia = require('rework-custom-media');

var varMap = reworkLoader.makeVarMap('./index.css');

var cssBundle = path.join('./', util.format('[name].%s.css', pkg.version));
var cssExtractTextPlugin = new ExtractTextPlugin(cssBundle, { allChunks: true });

module.exports = {
  resolve: {
    // if you want to @import stuff you installed with npm, such as suit css,
    // you probably need to include `style` in your `packageMains` config
    packageMains: [
      'webpack',
      'browser',
      'web',
      'node_modules',
      ['jam', 'main'],
      'style',
      'main'
    ]
  },
  entry: "./entry.js",
  output: {
    path: __dirname,
    filename: "bundle.js"
  },
  module: {
    loaders: [
      { test: /\.css$/, loader: ExtractTextPlugin.extract('style-loader', 'rework-webpack-loader') }
    ]
  },
  plugins: [
    new webpack.optimize.OccurenceOrderPlugin(),
    cssExtractTextPlugin
  ],
  rework: {
    use: [
     // I can't use initialize plugins here since webpack cli blows
      reworkCalc]
  }
};
```

Do you know if your loader works with the current webpack cli?

**Comment by aaronj1335** [comment]
well `@import` certainly won't work if you don't include `reworkLoader.plugins.imports`. try using the rework plugins used in the [README](https://github.com/aaronj1335/rework-webpack-loader/blob/master/README.md):

``` js
  rework: {
    use: [
      reworkLoader.plugins.imports,
      reworkLoader.plugins.urls,
      reworkLoader.plugins.stripLocalDefs(varMap),
      reworkCustomMedia({map: varMap}),
      reworkVars({map: varMap}),
      reworkCalc
    ]
  }
```

**Comment by priyatam** [comment]
Adding the plugins under rework throws the following error (as mentioned in my first comment):

```
ERROR in ./~/rework-webpack-loader!./index.css
Module not found: Error: Cannot resolve 'file' or 'directory' ./suitcss in ~/rework-demo
 @ ./~/rework-webpack-loader!./index.css 1:0-20
```

Do you have a working example?

**Comment by aaronj1335** [comment]
should be fixed by d4c5c5ed. please re-open if it doesn't work for you.

i've also added [an example directory](https://github.com/aaronj1335/rework-webpack-loader/tree/master/example) that might help.
